lazy_tensor_core/lazy_tensor_core/csrc/tensor_methods.cpp

```diff
@@ -1746,10 +1746,6 @@ void LazyTensor::min_out(LazyTensor& min, LazyTensor& min_indices,
   min.SetIrValue(ir::Value(node, 0));
   min_indices.SetIrValue(ir::Value(node, 1));
 }
-LazyTensor LazyTensor::mm(const LazyTensor& input, const LazyTensor& weight) {
-  return input.CreateFrom(
-      ir::ops::Dot(input.GetIrValue(), weight.GetIrValue()));
```

Comment:
presumably someone else is still using Dot class, but if not, we'd also want to remove that.  I didn't check..

Reply:
Yeah looks like at least two ops use it in tensor_methods.cpp. Added a TODO - lemme know if more stuff needs to happen here.